### DISCUSSION TITLE
Which version of Prometheus is compatible with 1.17.1 istio

### DISCUSSION THREAD

**Discussion by Yajnalatha** [istio/istio#44309] [Q&A]
Which version of Prometheus is compatible with 1.17.1 istio

**Comment by zirain** [comment]
I didn't think there's a strict compatible version between two projects

we currently use `19.6.1` in [addons](https://github.com/istio/istio/blob/master/samples/addons/prometheus.yaml)

**Comment by zirain** [comment]
closed it for now, feel free to reopen.
